metazoa/species/Actinia/Actinia_equina/Actinia_equina_GCA_011057435.1_assembly.md

```diff
@@ -2,7 +2,8 @@ Assembly
 --------
 
 The *Actinia equina* (_AE1_ strain) genome was sequenced at the [Centre for Genomic Research, University of Liverpool](https://www.liverpool.ac.uk/genomic-research/).
-Assembly  was undertaken using three separate assembly methods: CANU v1.7, SMARTdenovo or WTDBG [1] and submitted by the
-[Liverpool John Moores University](http://www.ljmu.ac.uk).
+
+Assembly  was undertaken using three separate assembly methods: CANU v1.7, SMARTdenovo, WTDBG and submitted by the
```

Comment:
Extra whitespace between 'Assembly' and 'was'

Reply:
Thanks for spotting, and I'll return eg-37.

Reply:
eg-37 is back, typo fixed